src/main/charts/bitbucket/values.yaml

```diff
@@ -777,6 +777,10 @@ tolerations: []
 #        topologyKey: topology.kubernetes.io/zone
 affinity: {}
 
+# -- Standard K8s schedulerName that will be applied to all Bitbucket pods
```

Comment:
Maybe link to https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#podspec-v1-core
or https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#specify-schedulers-for-pods